src/util/worker/mod.rs

```diff
@@ -367,4 +422,25 @@ mod test {
         }
         assert_eq!(rx.recv().unwrap(), 0);
     }
+
+    #[test]
+    fn test_on_tick() {
+        let mut worker = Builder::new("test-worker-tick").batch_size(4).create();
+        for _ in 0..10 {
+            worker.schedule("normal msg").unwrap();
+        }
+        thread::sleep(Duration::from_secs(3));
```

Comment:
Why sleep?

Reply:
It's for ensure `Scheduler` has already send all messages into task channel of `Worker`. So the `Worker` can always get a full batch when it call `fill_next_batch`.

Reply:
please add comment here

Reply:
I don't think it's necessary. Because runner is started at L435, so after L431, all 10 messages are still queued into the channel.